src/duplicacy_snapshotmanager.go

```diff
@@ -910,42 +927,73 @@ func (manager *SnapshotManager) CheckSnapshots(snapshotID string, revisionsToChe
 
                 var totalChunkSize int64
                 var uniqueChunkSize int64
+                var totalChunkCount int64
+                var uniqueChunkCount int64
+                var newChunkCount int64
+                var newChunkSize int64
 
                 for chunkID, _ := range chunks {
                     chunkSize := chunkSizeMap[chunkID]
                     totalChunkSize += chunkSize
+                    totalChunkCount += 1
+                    if earliestSeenChunks[chunkID] == snapshot.Revision {
+                        newChunkCount += 1
+                        newChunkSize += chunkSize
+                    }
                     if chunkUniqueMap[chunkID] {
                         uniqueChunkSize += chunkSize
+                        uniqueChunkCount += 1
                     }
                 }
 
-                files := ""
+                files := " \t "
                 if snapshot.FileSize != 0 && snapshot.NumberOfFiles != 0 {
-                    files = fmt.Sprintf("%d files (%s bytes), ", snapshot.NumberOfFiles, PrettyNumber(snapshot.FileSize))
+                    files = fmt.Sprintf("%d \t%s", snapshot.NumberOfFiles, PrettyNumber(snapshot.FileSize))
                 }
-                LOG_INFO("SNAPSHOT_CHECK", "Snapshot %s at revision %d: %s%s total chunk bytes, %s unique chunk bytes",
-                         snapshot.ID, snapshot.Revision, files, PrettyNumber(totalChunkSize), PrettyNumber(uniqueChunkSize))
+                creationTime := time.Unix(snapshot.StartTime, 0).Format("2006-01-02 15:04")
+                tagWithSpace := ""
+                if len(snapshot.Tag) > 0 {
+                    tagWithSpace = snapshot.Tag + " "
+                }
+                fmt.Fprintln(tableWriter, fmt.Sprintf(
```

Comment:
I think `snapshot id` and `revisions` should be the first two columns.

Reply:
I originally had them at the beginning, but I moved them to match "duplicity list" which puts them at the end. I'll move them back.